[2024-05-25T06:27:14.706+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T06:27:14.714+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 manual__2024-05-25T06:27:13.864326+00:00 [queued]>
[2024-05-25T06:27:14.717+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 manual__2024-05-25T06:27:13.864326+00:00 [queued]>
[2024-05-25T06:27:14.717+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-25T06:27:14.722+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_to_s3> on 2024-05-25 06:27:13.864326+00:00
[2024-05-25T06:27:14.724+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=66) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-25T06:27:14.724+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-05-25T06:27:14.725+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'upload_from_github_to_s3', 'download_and_upload_to_s3', 'manual__2024-05-25T06:27:13.864326+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/upload_from_github_to_s3_dag.py', '--cfg-path', '/tmp/tmpb8248dlg']
[2024-05-25T06:27:14.725+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask download_and_upload_to_s3
[2024-05-25T06:27:14.740+0000] {task_command.py:426} INFO - Running <TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 manual__2024-05-25T06:27:13.864326+00:00 [running]> on host 2fc9adb3db3b
[2024-05-25T06:27:14.765+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_from_github_to_s3' AIRFLOW_CTX_TASK_ID='download_and_upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T06:27:13.864326+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-25T06:27:13.864326+00:00'
[2024-05-25T06:27:14.765+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T06:27:15.492+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-05-25T06:27:15.628+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T06:27:15.629+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T06:27:15.633+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=upload_from_github_to_s3, task_id=download_and_upload_to_s3, run_id=manual__2024-05-25T06:27:13.864326+00:00, execution_date=20240525T062713, start_date=20240525T062714, end_date=20240525T062715
[2024-05-25T06:27:15.668+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T06:27:15.675+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-25T06:27:15.676+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
